packages/mesos/build

```diff
@@ -9,44 +8,38 @@ for patch in /pkg/extra/patches/*; do
   git -c user.name="Mesosphere CI" -c user.email="mesosphere-ci@users.noreply.github.com" am $patch
 done
 
-./bootstrap
-
 mkdir -p build
 pushd build
-# TODO(cmaloney): --with-glog=/usr --with-protobuf=/usr --with-boost=/usr
-# TODO(cmaloney): DESTDIR builds so we don't have to build as root?
-# TODO(nfnt): --with-grpc=/usr
-LDFLAGS="-Wl,-rpath,/opt/mesosphere/lib" "/pkg/src/mesos/configure" \
```

Comment:
Looks like we need to set `-DCMAKE_INSTALL_RPATH=/opt/mesosphere/lib` anyway. 
Not for the Mesos libraries, but for the non-bundled libraries that come installed by pkgpanda.

Docker executor fails with
```
mesos-docker-executor: error while loading shared libraries: libssl.so.1.1: cannot open shared object file: No such file or directory
```

Reply:
added